README.md

```diff
@@ -11,7 +11,7 @@ including detailed instructions on design, setup, and formatting.
 ## Quick Instructions
 
 1.  Do *not* fork this repository directly on GitHub.
-    Instead, please follow the instructions in [the setup instructions][setup]
+    Instead, please follow the instructions in [the setup instructions](http://swcarpentry.github.io/lesson-example/setup/)
```

Comment:
Please don't change the line 14 and instead change the line 95 to have the correct link.